### ISSUE TITLE
Facility upgrades appear in list twice

### ISSUE THREAD

**Issue by cwombat169** [ekolis/FrEee#298]
**Describe the bug**
Facility upgrades appear in list twice

**To Reproduce**
Load savegame look at upgrades available on the homeworld

**Expected behavior**
Upgrades should appear only once

**Actual behavior**
Upgrades appear twice

**Screenshots**
N/A

**Additional context**
See attached save

**Comment by ekolis** [comment]
Oh hey, this is a duplicate of #84. Did I ever fix that?
